scripts/build-logos.js

```diff
@@ -85,11 +76,13 @@ const buildLogos = async () => {
  */
 
 function createIndexJs(componentNames) {
-  return componentNames.map(name => `export { default as ${name} } from './${name}.js';`).join('\n');
+  const uniqueComponentNames = [...new Set(componentNames)];
```

Comment:
This prevents duplicate exports for icons which are used with multiple id's (e.g. Bancor)